SIT.Manager.Avalonia/Services/LocalizationService.cs

```diff
@@ -0,0 +1,119 @@
+            try
+            {
+                if (translations != null) App.Current.Resources.MergedDictionaries.Remove(translations);
+                App.Current.Resources.MergedDictionaries.Add(CreateResourceLocalization($"avares://SIT.Manager.Avalonia/Localization/{cultureInfo.Name}.axaml"));
+                _configService.Config.CurrentLanguageSelected = cultureInfo.Name;
+            }
+            catch // if there was no translation found for your computer localization give default English.
+            {
+                App.Current.Resources.MergedDictionaries.Add(CreateResourceLocalization($"avares://SIT.Manager.Avalonia/Localization/en-US.axaml"));
+                CultureInfo culture = new("en-US");
+                _configService.Config.CurrentLanguageSelected = cultureInfo.Name;
+            }
+        }
+
+        private ResourceInclude? resourceInclude;
+        private string currentLanguage = string.Empty;
+        /// <summary>
+        /// Changes the localization in .cs files that contains strings that you cannot change inside the page.
+        /// Functions contain neat parameters that help modify source strings, like in C#, but inside a Resource file.
+        /// Example will be: Your path is %1. %1 → path. | Output: Your path is: C:\Users\...
+        /// where % is the definition of parameter, and 1…n is the hierarchy of parameters passed to the function.
+        /// </summary>
+        /// <param name="key">string that you are accessing in Localization\*culture-info*.axaml file</param>
+        /// <param name="replaces">parameters in hierarchy, example: %1, %2, %3, "10", "20, "30" | output: 10, 20, 30</param>
+        public string TranslateSource(string key, params string[] replaces)
+        {
+            if (resourceInclude == null || string.IsNullOrEmpty(currentLanguage) || currentLanguage != _configService.Config.CurrentLanguageSelected)
+            {
+                try
+                {
+                    resourceInclude = CreateResourceLocalization($"avares://SIT.Manager.Avalonia/Localization/{_configService.Config.CurrentLanguageSelected}.axaml");
+                }
+                catch // If there was an issue loading current Culture language, we will default by English.
+                {
+                    resourceInclude = CreateResourceLocalization($"avares://SIT.Manager.Avalonia/Localization/en-US.axaml");
+                }
+            }
+
+            string result = "not found";
+            if (resourceInclude.TryGetResource(key, null, out object? translation))
+            {
+                if (translation != null)
+                {
+                    result = (string)translation;
+                    for (int i = 0; i < replaces.Length; i++)
+                    {
+                        result = result.Replace($"%{i + 1}", replaces[i]);
+                    }
+                }
+            }
+            return result;
+        }
```

Comment:
These two methods have significant overlap, Push some of the duplicate code to another function to avoid repeating the same thing. Also instead of using a try catch were reference the known available localizations to ensure it's valid before attempting